pkg/namespaceannoationfilter/filter.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package namespaceannoationfilter
```

Comment:
ughh, that's a very good long name indeed. Any idea making it more readable like dashes or making it shorter?

Reply:
Yes. It's a really bad name. Do you have better idea?

Reply:
what about making it belong to something more general like a universal `podfilter`?

Reply:
I moved it into the `selector`, (which contains a lot of `podfilter` and is really big :crying_cat_face: ).